src/types.ts

```diff
@@ -10,6 +10,9 @@ export interface ICustomizeOptions {
 export enum CustomizeRule {
   Match = "match",
   Append = "append",
+  AppendOnly = "append-only",
```

Comment:
I'm not sure this approach is healthy. Why not just assuming that it `append/prepend/replace-only` if there are no adjacent siblings with `match` rule?

Reply:
That could be a good way - technically I could detect if there's no `match` in a parent and then use the new behavior. I like the idea as it doesn't grow the API surface and it's a simple rule to explain.